solidity/scripts/install.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
@Shadowfiend thoughts on placing this installation script under repo's root directory in `/scripts/install.sh`?

Reply:
I'm down if that's where it is in other repos.

Reply:
Yep, that's why I proposed to move it. Cool, cool!